components/kubeconfig-service/pkg/transformer/kubeconfig.go

```diff
@@ -49,6 +49,5 @@ users:
       - get-token
       - "--oidc-issuer-url={{ .OIDCIssuerURL }}"
       - "--oidc-client-id={{ .OIDCClientID }}"
-      - "--oidc-client-secret={{ .OIDCClientSecret }}"
```

Comment:
Feel free to remove the corresponding config item from the chart, and https://github.com/kyma-project/control-plane/blob/main/components/kubeconfig-service/pkg/env/env.go